baseplate/experiments/__init__.py

```diff
@@ -184,13 +197,36 @@ def variant(self, name, user=None, bucketing_event_override=None,
                 logged_in=inputs.get('logged_in'),
                 cookie_created_timestamp=inputs.get('cookie_created_timestamp'),
                 app_name=inputs.get('app_name'),
+                event_type=EventType.BUCKET,
             )
             self._already_bucketed.add(bucketing_id)
 
         return variant
 
+    def expose(self, experiment_name, variant_name, user=None, **kwargs):
+
+        experiment = self._get_experiment(experiment_name)
+
+        if experiment is None:
+            return
+
+        inputs = dict(kwargs)
+
+        if user:
+            inputs.update(user.event_fields())
+
+        self._event_logger.log(
+            experiment=experiment,
+            variant=variant_name,
+            user_id=inputs.get('user_id'),
+            logged_in=inputs.get('logged_in'),
+            cookie_created_timestamp=inputs.get('cookie_created_timestamp'),
+            app_name=inputs.get('app_name'),
+            event_type=EventType.EXPOSE,
```

Comment:
Does _something_ need to be updated to account for this new field?

Reply:
Not in here. The EventLoggers that use this will need to be updated, but they're not in this repo.